vignettes/estimate_ascertainment.Rmd

```diff
@@ -20,128 +20,100 @@ editor_options:
 
-This vignette outlines the function within _cfr_ used to estimate the
-proportion of cases, infections or hospitalisations ascertained.
+::: {.alert .alert-warning}
+New to calculating disease severity using _cfr_? You might want to see the ["Get started" vignette first](cfr.html).
+:::
 
-First, we load the key packages required for this vignette:
+::: {.alert .alert-primary}
+## Use case {-}
+
+We wish to estimate one of the typical severity quantities used in epidemiology -- the *case fatality risk (CFR)*, *infection hospitality risk (IFR)* or  *hospitalisation fatality risk (HFR)* -- using a method that uses a time series of cases, infections or hospitalisations (respectively) and deaths, _while correcting for the delay in reporting the outcomes of cases_.
+:::
+
+::: {.alert .alert-secondary}
+## What we have {-}
+
+* A time-series of cases, hospitalisations or some other proxy for infections over time;
+* A time-series of deaths;
+* A delay distribution, describing the probability an individual will die $t$ days after they were initially infected.
+:::
+
+<!-- This can lead to biases due to some infections either being asymptomatic or having a pre-symptomatic phase.
+
+For example, during the recent COVID-19 pandemic, the U.K. did not roll out widespread testing until around May 2020.
+This means that during March and April, infections were heavily under-ascertained.
+We focus on this period within this vignette, as it illustrates the use for this analysis pipeline well. -->
+
+This vignette shows how to use the `estimate_reporting()` function in _cfr_ to estimate the proportion of cases, infections or hospitalisations ascertained, and consequently, the extent of under-ascertainment or under-reporting.
+
+First load _cfr_ and packages to access and plot data.
 
 ```{r, message = FALSE, warning=FALSE, eval = TRUE}
+# load {cfr} and data packages
 library(cfr)
 library(epiparameter)
 library(covidregionaldata)
-```
 
-and the four optional packages, to run the multiple examples in the last 
-section of this vignette:
-
-```{r}
+# packages to wrangle and plot data
 library(dplyr)
 library(tidyr)
 library(purrr)
 library(scales)
-library(ggplot2)
 library(forcats)
+library(ggplot2)
 ```
 
-# Motivation
+## Ascertainment for the Covid-19 pandemic in the U.K.
 
-There are several reasons why cases, infections or hospitalisations and 
-deaths might be under-ascertained during an outbreak of an infectious disease. 
-Some examples include:
+The function `estimate_reporting()` from the _cfr_ package estimates the proportion of cases, infections, hospitalisations -- or whichever proxy for infections is provided -- which have been ascertained.
```

Comment:
If we agree that the function is about estimating the ascertainment of cases, which also englobes the issue of cases not being reported, maybe a clearer way to present it is to name it `estimate_ascertainment`?

Reply:
This is a good question and I'm converting it into an issue to get more feedback.